tests/meta/workflow/test_analysis.py

```diff
@@ -29,3 +31,25 @@ def test_get_slurm_qos_for_case(mocker, case_id: str, priority, expected_slurm_q
 
     # THEN the expected slurm QOS should be returned
     assert slurm_qos is expected_slurm_qos
+
+
+def test_gene_panels_correctly_added():
+    """Test get correct gene panel list"""
+
+    # GIVEN a case that has a gene panel included in the master list:
+    default_panels_included: List[str] = [GenePanelMasterList.get_panel_names()[0]]
+    master_list = GenePanelMasterList.get_panel_names()
+    # WHEN converting the gene panels between the default and the gene_panel_master_list:
+    list_of_gene_panels_used = MipAnalysisAPI.convert_panels("cust002", default_panels_included)
+
+    # THEN the list_of_gene_panels_used should return all gene panels:
+    assert set(list_of_gene_panels_used) == set(master_list)
+
+    # GIVEN a case that has a gene panel that is NOT included in the master list:
+    default_panels_not_included: List[str] = ["ANOTHER_PANEL"]
+
+    # WHEN converting the gene panels between the default and the gene_panel_master_list:
+    list_of_gene_panels_used = MipAnalysisAPI.convert_panels("cust002", default_panels_not_included)
+
+    # THEN the list_of_gene_panels_used should return the custom panel and OMIM-AUTO:
+    assert set(list_of_gene_panels_used) == set(default_panels_not_included + ["OMIM-AUTO"])
```

Comment:
I think you should use the new shiny constant class you just created instead of hard-coding `OMIM_AUTO`